### PR TITLE
Fix: Prevent loading an undef variable instance, get preview scene on init

### PR DESCRIPTION
**Pull request by bryce-seifert**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -398,6 +397,12 @@ instance.prototype.updateScenesAndSources = async function() {
 		self.scenes[scene.name] = scene;
 	}
 
+	if (self.states['studio_mode'] = true) {
```
Comment (krocheck) [inline index.js:400]: This would need to be an `==` or `===`

Comment (bryce-seifert) [reply index.js]: Thank you for catching, fixed.

Comment (krocheck) [reply index.js]: please re-test

Comment (bryce-seifert) [reply index.js]: I re-tested upon change, and it still works as expected

**Pull request merged**